Release automation fails at the last step

We've enabled some automatic handling of the staging repositories, so the last step in the workflow is not needed.

```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':closeAndReleaseRepository'.
> No matching staging repository found in account ***. You can can explicitly choose one by passing it as an option like this "./gradlew closeAndReleaseRepository --repository=comexample-123". Available repositories are:
```

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -23,7 +23,6 @@ jobs:
         run: |
           ./gradlew clean --no-daemon
           ./gradlew publish --no-daemon --no-parallel
-          ./gradlew closeAndReleaseRepository
         env:
           ORG_GRADLE_PROJECT_mavenCentralUsername: ${{ secrets.MAVEN_CENTRAL_USERNAME}}
           ORG_GRADLE_PROJECT_mavenCentralPassword: ${{ secrets.MAVEN_CENTRAL_PASSWORD}}

